spec/models/lockstep/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Lockstep::User' do
+
+  context 'when calling feature flag' do
```

Comment:
Why `when calling feature flag` I see no connection 😕

Reply:
Sorry my bad, will resolve this.